packages/manager/modules/pci/src/components/project/billing/hourly-resource-list/controller.js

```diff
@@ -1,20 +1,23 @@
 
-export default /* @ngInject */ function (
-  DetailsPopoverService,
-) {
-  const self = this;
+export default /* @ngInject */ class {
+  /* @ngInject */
+  constructor(DetailsPopoverService) {
```

Comment:
i'm not used to see uppercased variable names outside of class declaration, did i miss something?